module-domain/src/main/kotlin/site/yourevents/invitation/service/InvitationService.kt

```diff
@@ -26,7 +26,8 @@ class InvitationService(
             InvitationVO.from(
                 InvitationVO(
                     member = member,
```

Comment:
VO 객체에서 VO 객체를 생성하는 로직이 중복된 것 같습니다! 저번 PR때 미처 확인하지 못한 부분이군요..ㅎㅎ
```suggestion
            InvitationVO(
                    member = member,
```

Reply:
아 그렇군요!! 제가 from의 매개변수에 간단하게 처리를 하고 싶어 InvitationVO를 한번 더 넣었더니 이런 중복이 발생했던 것 같습니다!! 동일하게 InvitaitonInformationVO과 InvitationThumbnailVO도 중복이 일어나기에 다같이 수정하였습니다!! 감사합니다.